tools/release_do_nothing.py

```diff
@@ -0,0 +1,758 @@
+) -> None:
+    _break_print("Read the Docs checks ...")
+
+    message = (
+        "Visit https://readthedocs.org/projects/scitools-iris/versions/ and "
+        "make sure you are logged in."
+    )
+    _wait_for_done(message)
+
+    message = f"Set {release_strings.tag} to Active, un-Hidden."
+    _wait_for_done(message)
+
+    message = f"Set {release_strings.branch} to Active, Hidden."
+    _wait_for_done(message)
+
+    message = (
+        "Consider deactivating older branch doc builds - those formatted 0.0.x"
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Visit https://scitools-iris.readthedocs.io/en/{release_strings.tag} "
+        "to confirm:\n\n"
+        "- The docs have rendered.\n"
+        f"- The version badge in the top left reads: {release_strings.tag} .\n"
+        "   (this demonstrates that setuptools_scm has worked correctly).\n"
+        "- The What's New looks correct.\n"
+        f"- {release_strings.tag} is available in RTD's version switcher.\n"
+    )
+    if not is_release_candidate:
+        message += (
+            "- Selecting 'stable' in the version switcher also brings up the "
+            f"{release_strings.tag} render."
+        )
+    _wait_for_done(message)
+
+    message = (
+        f"Visit https://scitools-iris.readthedocs.io/en/{release_strings.branch} "
+        "to confirm:\n\n"
+        "- The docs have rendered\n"
+        f"- The version badge in the top left includes: {release_strings.branch} .\n"
+        f"- {release_strings.branch} is NOT available in RTD's version switcher.\n"
+    )
+    _wait_for_done(message)
+
+
+def check_pypi(
+    release_strings: ReleaseStrings, is_release_candidate: bool
+) -> str:
+    _break_print("PyPI checks ...")
+    _break_print("If anything goes wrong, manual steps are in the documentation.")
+
+    message = (
+        "Confirm that the following URL is correctly populated:\n"
+        f"https://pypi.org/project/scitools-iris/{release_strings.release}/"
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Confirm that {release_strings.release} is at the top of this page:\n"
+        "https://pypi.org/project/scitools-iris/#history"
+    )
+    _wait_for_done(message)
+
+    if is_release_candidate:
+        message = (
+            f"Confirm that {release_strings.release} is marked as a "
+            f"pre-release on this page:\n"
+            "https://pypi.org/project/scitools-iris/#history"
+        )
+    else:
+        message = (
+            f"Confirm that {release_strings.release} is the tag shown on the "
+            "scitools-iris PyPI homepage:\n"
+            "https://pypi.org/project/scitools-iris/"
+        )
+    _wait_for_done(message)
+
+    message = (
+        f"Visit the below and click `view hashes`:\n"
+        f"https://pypi.org/project/scitools-iris/{release_strings.release}#files\n"
+    )
+    sha256 = _get_input(message, "Input the SHA256 hash")
```

Comment:
Input the SHA256 hash from the topmost link (I can't remember which build it was, specifying would be helpful)

Reply:
6940d28af